### ISSUE TITLE
Splash Screen Image doesn't fit across devices with different Android Versions

### ISSUE THREAD

**Issue by MikeP0911** [dotnet/maui#20605]
### Description

I have used a custom image for splash screen. It is a *.png image. The image appears as expected in Android 12 Mobile Phones. But it is enlarged in Android 11 Mobile Phone and Harmony OS tablet.

### Steps to Reproduce

1. Create a new  MAUI project
2. Delete the default splash image from Resources/Splash folder
3. Add a new splash image in the Resources/Splash folder
4. Select MauiSplashScreen for BuildAction in the File Properties
5. Build and Deploy the project in Android 11 and Android 12 mobile phones

### Version with bug

7.0.101

### Is this a regression from previous behavior?

Yes, this used to work in Xamarin.Forms

### Last version that worked well

7.0.101

### Affected platforms

Android

### Affected platform versions

Android 11, Android 12

**Comment by SmartmanApps** [comment]
Yeah I just recently noticed differences on different Android devices (mostly because circumstances meant I had to use a new one). One one emulator I have my logo appearing in a stripe across the screen, but on another one it's in a circle. I'm not sure if this is the same issue being discussed here - is it meant to be like that?

**Comment by PureWeen** [comment]
@MikeP0911  @SmartmanApps can you include screen shots?

**Comment by SmartmanApps** [comment]
@PureWeen 
No probs. Pixel 3a API 28 and Pixel 4 API 34
![image](https://github.com/dotnet/maui/assets/20635275/e528a3e3-b5f5-4220-9def-57c39a5a50d4)
![image](https://github.com/dotnet/maui/assets/20635275/d206e409-901e-4881-bb12-862f00b388cc)

**Comment by MikeP0911** [comment]
Attaching images from two different android phones.
This is the expected screen (Sony Experia XQ-BT44 - Android 12)
![SplashIcon-Actual](https://github.com/dotnet/maui/assets/117801555/1da842fc-f74b-450a-8a04-406e2ac187fc)


The below screenshot is from another android mobile phone which is expanded.(Fujitsu Arrows F-02L-  Android 11)
![SplashIcon-Expanded](https://github.com/dotnet/maui/assets/117801555/1f717cc4-e5e0-40ef-81ad-ec00537f4186)

**Comment by MikeP0911** [comment]
Awaiting solution

**Comment by MikeP0911** [comment]
@PureWeen, 
Waiting for the reply, please update. Thanks

**Comment by MikeP0911** [comment]
Any alternative solution or any working sample available for reference for the above issue? 
Kindly share if any, since it's been a very long time. Thanks.

**Comment by MikeP0911** [comment]
Hi,

We are awaiting a solution for this issue. It's been open for quite a long time and there are no updates.

We appreciate a solution or any updates on this issue. Thanks.

**Comment by QianaJiao** [comment]
Verified this issue with Visual Studio 17.10.0 preview2 (8.0.14). Can repro on Android platform.

**Comment by MikeP0911** [comment]
@PureWeen , Any solutions on this issue will be helpful. Thanks

**Comment by PureWeen** [comment]
@mattleibow thoughts?

**Comment by MikeP0911** [comment]
@PureWeen @mattleibow any updates on this issue. It's been almost two months.

Thanks

**Comment by James-McMenamin** [comment]
still thing.

**Comment by bcaceiro** [comment]
Still happening in latest dotnet maui versions ( 8 and 9)

**Comment by anpin** [comment]
My workaround was to remove the image and make startup faster 🙂

**Comment by bcaceiro** [comment]
@anpin well, removing the image is not a valid workaround eheheh

**Comment by jfversluis** [comment]
To be honest I'm not sure if there is anything to fix here. This just seems Android behavior as of a certain Android version: https://github.com/dotnet/maui/issues/25500#issuecomment-2437297261

@jonpryor @jonathanpeppers @Redth do any of the Jonathans know?

**Comment by ginxFromYt** [comment]
I have encountered this issue when i tried to debug on physical Android 10.0 - API 29, but is scaling properly on android 13 -api 33.

`
<MauiSplashScreen Include="Resources\Splash\logo.png" Color="White" BaseSize="1437,1632">
  <CopyToOutputDirectory>Always</CopyToOutputDirectory>
</MauiSplashScreen>`

![Image](https://github.com/user-attachments/assets/43ed47b2-a602-4c32-b7cc-97015d532ae6)

**Comment by ginxFromYt** [comment]
Good day! Here is how I fix the issue along with Co-Pilot 🙂

I first created a bunch of dedicated splash screen images with diff. sizes in a dedicated screen density for android 
`
Platforms/
└── Android/
    └── Resources/
        ├── mipmap-mdpi/
        │   └── ic_splash.png (48x48)
        ├── mipmap-hdpi/
        │   └── ic_splash.png (72x72)
        ├── mipmap-xhdpi/
        │   └── ic_splash.png (96x96)
        ├── mipmap-xxhdpi/
        │   └── ic_splash.png (144x144)
        └── mipmap-xxxhdpi/
            └── ic_splash.png (192x192)
`

and then created an new xml file under android platform 

#Platforms\Android\Resources\drawable\splash.xml
`
<?xml version="1.0" encoding="utf-8"?>
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
  <item android:drawable="@color/splash_background" /> 
  <item>
    <bitmap
        android:gravity="center"
        android:src="@mipmap/ic_splash" />
  </item>
</layer-list>

`

#Platforms\Android\Resources\values\colors.xml
`
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <color name="colorPrimary">#512BD4</color>
    <color name="colorPrimaryDark">#2B0B98</color>
    <color name="colorAccent">#2B0B98</color>
    <color name="splash_background">#F5E9F1</color>
  
 
</resources>
`

I also created a new xml file 
#Platforms\Android\Resources\values\styles.xml

`
<resources>
  <style name="Theme.SplashScreen" parent="Theme.AppCompat.Light.NoActionBar">
    <item name="android:windowBackground">@drawable/splash</item>
  </style>
  <style name="Maui.SplashTheme" parent="Theme.SplashScreen">
    <item name="android:windowBackground">@drawable/splash</item>
  </style>
</resources>
`

also don't forget to add android:theme="@style/Maui.SplashTheme" to you AndroidManifest.xml file
`<?xml version="1.0" encoding="utf-8"?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android">
	<application android:allowBackup="true" android:icon="@mipmap/icon" android:supportsRtl="true" android:theme="@style/Maui.SplashTheme"></application>
	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
	<uses-permission android:name="android.permission.INTERNET" />
	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
	<uses-permission android:name="android.permission.CAMERA" />
</manifest>`

![Image](https://github.com/user-attachments/assets/8ff8b47b-8f26-43ef-a55b-ff68a221fcc1)

these approach worked for me in Android 10 - API 29

![Image](https://github.com/user-attachments/assets/80e0d0a2-4a8c-4841-a783-970f3b813f2d)

**Comment by bharathwajv** [comment]
Same issue for me as well, Why still no resolution of fix given from Microsoft end. MAUI - Microsoft hobby project
